test/test_helper.rb

```diff
@@ -9,6 +9,7 @@
   STDERR.puts 'Running without SimpleCov'
 end
 
+require 'pry'
```

Comment:
This was actually a great suggestion by @bf4 - if you `require 'pry'` it will load any pry plugins for you. So if you are non-MRI you will only get pry (works across rubies), if you are MRI you get `pry-byebug` as well (both specified in Gemfile).